### PR TITLE
Add support for creating policies

### PR DESCRIPTION
**Pull request by samuel4x4**
Request JSON body:

`{
    "policy": {
        "type": "application/json",
        "blob": "{'foobar_user': 'role:compute-user'}"
    }
}`

### CODE REVIEW

**Comment by samuel4x4** [comment]
@haphan

File: src/Identity/v3/Api.php [hunk 1]
```diff
@@ -772,6 +772,7 @@ public function postPolicies(): array
         return [
             'method' => 'POST',
             'path'   => 'policies',
+            'jsonKey' => 'policy',
```
Comment (jamiehannaford) [inline src/Identity/v3/Api.php]: super minor, but can we realign the keys?

Comment (samuel4x4) [inline src/Identity/v3/Api.php]: @jamiehannaford yes, sure ;) Is it possible to merge this PR and tag a new release? I'm waiting also for #120 ;) Thanks!

**Comment by jamiehannaford** [comment]
[v2.0.9](https://github.com/php-opencloud/openstack/releases/tag/v2.0.9) is now live

**Comment by samuel4x4** [comment]
thanks ;)

**Pull request merged**